store/acistore/store.go

```diff
@@ -97,39 +96,37 @@ func (e *StoreRemovalError) Error() string {
 	return s
 }
 
-// Store encapsulates a content-addressable-storage for storing ACIs on disk.
-type Store struct {
-	dir       string
-	stores    []*diskv.Diskv
-	db        *db.DB
-	treestore *TreeStore
+// ACIStore encapsulates a content-addressable-storage for storing ACIs on disk.
+type ACIStore struct {
```

Comment:
I also prefer avoiding package vs functions redundancy.

I think Store will be enough..
Probably also `NewStore` or just `New` (since there's only one New function) should be ok